test/paramSpec.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var paramFn = require('../src/param');
+
+describe('param', function() {
+  describe('param', function() {
```

Comment:
This is very nit picky but how do you feel about changing the outer describe to something other than `param` so it's more descriptive when looking through the test results? I noticed it in the update file as well.